### ISSUE TITLE
Double-clicking `install_sct.bat` results in the Command Prompt window disappearing

### ISSUE THREAD

**Issue by joshuacwnewton** [spinalcordtoolbox/spinalcordtoolbox#3762]
As per @RignonNoel's comment in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3761#discussion_r849571477,  Windows users should be able to double-click `install_sct.bat` to run the installer without having to use the command line. But, currently, we're only testing the terminal-based installation. So, there are a few issues that were missed:

1. This line uses a [`echo|set /p` trick](https://stackoverflow.com/a/7105690) to be able to print the Y/N prompt from the `rmdir` command on the same line as the previous echo:

    https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/61db54c410366685c109e0d399f1d528c9704e23/install_sct.bat#L19-L22

    Without it:

    ```
    ### Continuing will overwrite the existing installation directory
    spinalcordtoolbox\, Are you sure (Y/N)?
    ```

    With it:

    ```
    ### Continuing will overwrite the existing installation directory spinalcordtoolbox\, Are you sure (Y/N)?
    ```

    However, when double-clicking the `install_sct.bat` file, the command window disappears right after this line executes. (And, if we do a normal `echo`, the command window stays open, which is what we want.)

    It would be nice to find a way to get this effect while still enabling a double-click install.

2. When the installation finishes, the command window closes automatically. So, we should probably add a `pause` statement here, before the exit:

    https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/61db54c410366685c109e0d399f1d528c9704e23/install_sct.bat#L92

**Comment by joshuacwnewton** [comment]
Whoops! It looks like https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3762 wasn't linked to this issue. Doing that now.

**Comment by RignonNoel** [comment]
> Whoops! It looks like #3762 wasn't linked to this issue. Doing that now.

3762 is the issue on which you created the comment @joshuacwnewton. Did you made the comment on the wrong issue or does I miss something ? 😆

**Comment by joshuacwnewton** [comment]
Oops, I meant https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3778. :P 

But, basically, when https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3778 was merged, this issue didn't get closed, because we wrote "#3762" instead of "Closes #3762".
